elastio-nat-provision-lambda/requirements.txt

```diff
@@ -0,0 +1,7 @@
+boto3==1.34.139
+botocore==1.34.139
+jmespath==1.0.1
+python-dateutil==2.9.0.post0
+s3transfer==0.10.2
+six==1.16.0
+urllib3==2.2.2
```

Comment:
From what I see in code only `boto3` and `botocore` are used. What are those other dependencies?

Reply:
it's an output of `pip freeze` which creates something like a Cargo's lock file